config.json

```diff
@@ -315,6 +316,8 @@
     1065659890036646019
   ],
   "insiderAnnouncementChannel": 1043898319883219004,
+  "insiderInfoChannel": 1279201622651572317,
+  "insidersChannel": 187649467611086849,
```

Comment:
This is a duplicate of `insiderCommandLockedToChannel`

Reply:
`insidersChannel` is a better name so lets consolidate older usage into that name but still accept `insiderCommandLockedToChannel` an alias for both. Might require some custom logic in ConfigJson struct

Reply:
How do you want this to work? Accept both values in config, but require it to be referenced as `Program.cfgjson.InsidersChannel`?

Reply:
One variable thats used but that accepts two json names.. probably done by setting both variables and then setting one to the other. It sounds annoying but makes more sense when configuring